pptx_blueprint/__init__.py

```diff
@@ -33,14 +33,47 @@ def replace_text(self, label: str, new_text: str) -> None:
         for shape in shapes:
             shape.text = new_text
 
-    def replace_picture(self, label: str, filename: _Pathlike) -> None:
+    def replace_picture(self, label: str, filename: _Pathlike, *, do_not_scale_up: bool = False) -> None:
         """Replaces rectangle placeholders on one or many slides.
 
         Args:
             label (str): label of the placeholder (without curly braces)
             filename (path-like): path to an image file
+            do_not_scale_up (bool): deactivates that the image is enlarged (default: False)
         """
-        pass
+        shapes_to_replace = self._find_shapes(label)
```

Comment:
The method signature of `_find_shapes` will be changed in #7. Maybe want to change this call to reflect that change?

Reply:
I have updated the usage of `_find_shapes()`. The same two lines of code are used that can be already be found at `replace_text()`.